administration/src/errors/getMessageFromApolloError.tsx

```diff
@@ -16,7 +16,7 @@ const getMessageFromApolloError = (error: ApolloError): GraphQLErrorMessage => {
   }
 
   const codesEqual = error.graphQLErrors.every(
-    (value, index, array) => value.extensions.code === array[0].extensions.code
+    (value, index, array) => value.extensions?.code === array[0].extensions?.code
```

Comment:
If there are runtime errors in the backend there will be no code in the return message (even though an error occured)